decimal64math_test.go

```diff
@@ -379,3 +379,93 @@ func BenchmarkDecimal64Sub(b *testing.B) {
 		_ = x[i%len(x)].Sub(y[i%len(y)])
 	}
 }
+
+func TestAddOverflow(t *testing.T) {
+	require := require.New(t)
+
+	a := MustParseDecimal64("-9.999999999999999e384") // largest dec64 number
+	b := MustParseDecimal64("0.000000000000001e384")
+	require.Equal(NegInfinity64, a.Sub(b))
+
+	a = MustParseDecimal64("9.999999999999999e384") // largest dec64 number
+	b = MustParseDecimal64("0.000000000000001e384")
+	require.Equal(Infinity64, a.Add(b))
+
+	a = MustParseDecimal64("9.999999999999999e384") // largest dec64 number
+	b = MustParseDecimal64("0.0000000000000001e384")
+	require.Equal(a, b.Add(a))
+
+	a = MustParseDecimal64("9.999999999999999e384") // largest dec64 number
+	b = MustParseDecimal64("0")
+	require.Equal(a, b.Add(a))
+
+}
+
+func TestQuoOverflow(t *testing.T) {
+	require := require.New(t)
+
+	max := MustParseDecimal64("1e384") // largest dec64 number
```

Comment:
Not the largest. Ditto below.

Reply:
Actually, we should simply define `const Max64 = newFromParts(…)` in decimal64const.go. Also `Min64` (the smallest positive subnormal) and maybe even `MaxSubnormal64` and `MinNormal64`